src/runtime/google-analytics-event-listener.js

```diff
@@ -77,4 +78,20 @@ export class GoogleAnalyticsEventListener {
       });
     }
   }
+
+  /**
+   * Function to determine whether publisher is eligible for GA.
+   * @returns {boolean}
+   */
+  static isGaEligible() {
```

Comment:
We want to pass `deps` into this, dont we?

Reply:
Yep, just updated. Sorry for the delay.